index.html

```diff
@@ -1,13 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vanilla Todo</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <div class="container"></div>
-  </body>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Vanilla Todo</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="container">
+    <h2> 대기중(
+      <span id="pendingTasksNumber">0</span>
+      )
+    </h2>
+    <ul id="todosPendingList">
+
+    </ul>
+  
+    <h2> 완료됨(
+      <span id="finishedTasksNumber">0</span>
+      )
+    </h2>
+    <ul id = "todosFinishedList">
+    
+        
+    </ul>
+  
+    <form class="inputNewTasks">
+      <input id="todoInput" type="text" size="20" placeholder="새로운 할 일 추가">
+      <button id="addButton" type="submit">+</button>
```

Comment:
조금더 명확한 네이밍을 사용하시면 좋을 것 같아요. 제출하는 버튼인것은 태그명을 통해 알 수 있으니, 어떤 "기능"인지가 돋보이도록 `todo-submit`은 어떨까요?

```suggestion
      <button id="todo-submit" type="submit">+</button>
```